isis/appdata/import/cassis.lbl.tpl

```diff
@@ -20,94 +20,93 @@ Object = IsisCube
-    PredictedRepetitionFrequency = 97605.7 <ms>
-    GroundTrackVelocity          = 0.5735 <km/s>
-    ForwardRotationAngle         = 158.600 <deg>
-    SpiceMisalignment            = 1.235 <deg>
-    FocalLength                  = 0.8770 <m>
-    FNumber                      = 6.50
-    ExposureTimeCommand          = 300
-    FrameletNumber               = 0
-    NumberOfFramelets            = 30
-    ImageFrequency               = 4000000 <ms>
-    NumberOfWindows              = 6
-    UniqueIdentifier             = 100731832
-    UID                          = 100731832
-    ExposureTimestamp            = 2f014e8bf23d21d1
-    ExposureTimePEHK             = 2.880e-003 <ms>
-    PixelsPossiblySaturated      = 0.00
-    IFOV                         = 1.140e-005
-    IFOVUnit                     = rad/px
-    FiltersAvailable             = "BLU RED NIR PAN"
-    FocalLengthUnit              = M
-    TelescopeType                = "Three-mirror anastigmat with powered fold
-                                    mirror"
-    DetectorDescription          = "2D Array"
-    PixelHeight                  = 10.0
-    PixelHeightUnit              = MICRON
-    PixelWidth                   = 10.0
-    PixelWidthUnit               = MICRON
-    DetectorType                 = "SI CMOS HYBRID (OSPREY 2K)"
-    ReadNoise                    = 61.0
-    ReadNoiseUnit                = ELECTRON
-    MissionPhase                 = MCO
-    SubInstrumentIdentifier      = 61.0
-    WindowCount                  = 0
-    Window1Binning               = 0
-    Window1StartSample           = 0
-    Window1EndSample             = 2047
-    Window1StartLine             = 354
-    Window1EndLine               = 633
-    Window2Binning               = 0
-    Window2StartSample           = 0
-    Window2EndSample             = 2047
-    Window2StartLine             = 712
-    Window2EndLine               = 966
-    Window3Binning               = 0
-    Window3StartSample           = 0
-    Window3EndSample             = 2047
-    Window3StartLine             = 1048
-    Window3EndLine               = 1302
-    Window4Binning               = 0
-    Window4StartSample           = 0
-    Window4EndSample             = 2047
-    Window4StartLine             = 1409
-    Window4EndLine               = 1662
-    Window5Binning               = 0
-    Window5StartSample           = 640
-    Window5EndSample             = 767
-    Window5StartLine             = 200
-    Window5EndLine               = 208
-    Window6Binning               = 0
-    Window6StartSample           = 1280
-    Window6EndSample             = 1407
-    Window6StartLine             = 1850
-    Window6EndLine               = 1858
-    YearDoy                      = 2016327
-    ObservationId                = CRUS_049185_238_0
+    {%if exists("Product_Observational.Identification_Area.Alias_List")%}ObservationId             = {{Product_Observational.Identification_Area.Alias_List.Alias.alternate_id}}{%endif%}
```

Comment:
I can clean this up by not having everything in one line. I originally thought this would create newlines in the isis label. But I found out that these are handled when the isis label is generated. Since it will take a bit to clean this, I just left it as one line for now.

Reply:
I do think it would be easier to read if it's not on one line. It's really good to now that it's possible to do that without adding newlines to the label.